.devcontainer/devcontainer.json

```diff
@@ -14,7 +14,7 @@
   "portsAttributes": {
     "3000": {
       "label": "Application",
-      "onAutoForward": "openPreview"
+      // "onAutoForward": "openPreview"
```

Comment:
Can we remove this if we're no longer using it?

Reply:
Yep. Just deleted it.